src/components/date-picker/pickers/MonthPicker.tsx

```diff
@@ -123,7 +125,8 @@ export class MonthPicker extends Picker {
         return (
             <div
                 className="datepicker-month"
-                onClick={() => { // tslint:disable-line:jsx-no-lambda prettier
+                // eslint-disable-next-line react/jsx-no-bind
+                onClick={() => {
```

Comment:
We really should fix this one, as we don't want people reading these docs to copy this behaviour. I'll probably fix it, so view this mostly as a "note to self".